modules/nw-metallb-configure-community-bgp-advertisement.adoc

```diff
@@ -0,0 +1,111 @@
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-BGP-advertisement-community-alias_{context}"]
+= Configuring MetalLB with a BGP advertisement and community alias
+
+Configure MetalLB as follows so that the IP address pool is advertised with the BGP protocol and the community alias set to the numeric value of the well-known `NO_ADVERTISE` community.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+== Example: Advertise an address pool configuration with BGP
+
+In the following example, the peer BGP router `doc-example-peer-community` receive one `203.0.113.200/32` route and one `fc00:f853:ccd:e799::1/128` route for each load-balancer IP address that MetalLB assigns to a service.
+A community alias is configured with the `NO_ADVERTISE` community.
+
+.Procedure
+
+. Create an IPAddressPool.
+
+.. Create a file, such as `ipaddresspool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example-bgp-community
+spec:
+  addresses:
+    - 203.0.113.200/30
+    - fc00:f853:ccd:e799::/124
+----
+
+.. Apply the configuration for the IPAddressPool:
++
+[source,terminal]
+----
+$ oc apply -f ipaddresspool.yaml
+----
+
+. Create a community alias named `community1`. 
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: Community
+metadata:
+  name: community1
+  namespace: metallb-system
+spec:
+  communities:
+    - name: NO_ADVERTISE
+    - value: '65535:65282'
+----
+
+. Create a BGP peer named `doc-example-peer-community`. 
+
+.. Create a file, such as `bgppeer.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: BGPPeer
+metadata:
+  namespace: metallb-system
+  name: doc-example-peer-community
```

Comment:
Do you have to name bgp peer as "doc-example-peer-community"? Can it be just doc-example-bgp-peer instead?